src/pages/patterns/status-indicator-pattern/index.mdx

```diff
@@ -509,8 +516,8 @@ our system.
 #### Color
 
 Differential indicators are either displaying a positive or a negative value.
-Color is optional in these situations as long as the value has either a ‘+’ or
-‘-’ in front of it, a chevron icon, or an arrow icon. Unless the data involves
+Color is optional in these situations as long as the value has either a "+" or
```

Comment:
The quotes here are coming in correctly here.
<img width="100" alt="image" src="https://github.com/user-attachments/assets/7a20f875-b078-4e7a-a046-94a9a418a767">